docs/file/lib/tupelo.js.html

```diff
@@ -477,10 +485,26 @@
  *
  * @return {TupeloClient} Tupelo client connection.
  */
-function connect(walletServer, walletCreds) {
```

Comment:
Whats the reason you opted for `connectTLS` and `connectInsecure` vs having a third argument to `connect` like `TupeloClient`?

Reply:
At first I wanted the api to be explicit about whether or not you were connecting securely to the rpc server, so that's where the two functions came from. Adding the third argument to connect came later during the actual implementation.

My main concern is that no one connects insecurely by mistake. I think adding a third argument which defaults to a secure connection and requires an explicit override accomplishes that goal as well, so I'm ok with making that change.

Reply:
Yeah that makes sense - I think either way is good, both approaches have their merits. I was mainly thinking that in practice I would control this with an environment variable or a config file, and just having those setting passed to the same function seems a bit easier than doing an if block. At that point the user is ultimately only dealing with a single toggle regardless of how the underlying function is constructed. But again, I think 2 explicit functions approach here also has its own benefits